app/src/main/java/edu/mills/cs115/fruitthief/database/Fruit.kt

```diff
@@ -0,0 +1,26 @@
+package edu.mills.cs115.fruitthief.database
+
+import androidx.room.ColumnInfo
+import androidx.room.Entity
+import androidx.room.PrimaryKey
+import java.util.*
+
+/**
+ * Data table of fruit and their seasons. Prepopulated when the user first opens the app.
```

Comment:
I wouldn't put in the KDoc that it's prepopulated when the user first opens the app. KDoc is supposed to be about the interface not about how other classes use it. 

You could link to the prepopulate method so clients know to call it.

Reply:
This comment has not been addressed.